roofit/roofitcore/inc/RooAbsOptTestStatistic.h

```diff
@@ -55,6 +55,8 @@ class RooAbsOptTestStatistic : public RooAbsTestStatistic {
   Bool_t isSealed() const { return _sealed ; }
   const char* sealNotice() const { return _sealNotice.Data() ; }
 
+  void setBatchMode(bool on = true);
```

Comment:
Add minimal doxygen comment: `/// <one sentence>.`

Reply:
Don't forget the `.` at the end for doxygen to know what's the "brief" and what's the extended part.

Reply:
Done!